client/js/screens/UserProfile/styles.js

```diff
@@ -1,6 +1,72 @@
 import {StyleSheet} from 'react-native';
 import theme from '../../config/theme';
 
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: '100%',
+    width: '100%',
+    padding: 30,
+  },
+  profileImage: {
+    width: 150,
+    height: 150,
+    borderRadius: 150,
```

Comment:
for images in react native use flexbox and resize mode properties to make it more responsive. 

https://medium.com/the-react-native-log/tips-for-react-native-images-or-saying-goodbye-to-trial-and-error-b2baaf0a1a4d